### ISSUE TITLE
Items stuck in unusable state

### ISSUE THREAD

**Issue by notsureawake** [shirsig/Cleanup#1]
Often when sorting with the default method, a few items become *stuck*, turning gray and unusable until logging out to char select and logging back in.  A /rl doesn't fix it.  If any other info is required please let me know.  This is with the current version of the addon on Elysium.

**Comment by shirsig** [comment]
Yeah I've seen it myself. seems to be a server bug. you can get rid of it by relogging. Only seems to happen when the inventory is in chaos though, not for resorting parts of it.

**Comment by phendrome** [comment]
What would be the best way to resolve this issue? Keeps happening. Contact the developers for the server?

**Comment by shirsig** [comment]
upvote this: https://elysium-project.org/bugtracker/issue/966

**Comment by notsureawake** [comment]
Would a configurable delay have the ability to solve this issue also?

**Comment by shirsig** [comment]
The delay needs to be pretty huge (like 100x slower). You can try changing the "break" on line 259 to "return" and increasing the delay on line 129 from .2 to .5 or something.

**Comment by notsureawake** [comment]
Alright, thanks very much for your help and the time you put in to your
addons.

**Comment by shinnova** [comment]
> The delay needs to be pretty huge (like 100x slower). You can try changing the "break" on line 259 to "return" and increasing the delay on line 129 from .2 to .5 or something.

After doing this I haven't experienced the issue at all anymore. Thank you for the workaround!

**Comment by calliberjoe** [comment]
im so confused when i opened up the lua file there is no delay thing anywhere, unless he meant the sortbags lua
